testing/moosefs/APKBUILD

```diff
@@ -0,0 +1,148 @@
+# Contributor: Michael Pirogov <vbnet.ru@gmail.com>
+# Maintainer: Michael Pirogov <vbnet.ru@gmail.com>
+pkgname="moosefs"
+pkgver="3.0.105"
+pkgrel=0
+pkgdesc="Open Source, Petabyte, Fault-Tolerant, Highly Performing, Scalable Network Distributed File System"
+url="https://moosefs.com"
+arch="all"
+license="GPL-2.0-only"
+makedepends="autoconf automake zlib-dev libpcap-dev libtool fuse-dev python3-dev"
+pkgusers="mfs"
+pkggroups="mfs"
+subpackages="$pkgname-doc $pkgname-client $pkgname-master $pkgname-chunkserver $pkgname-metalogger $pkgname-cgi::noarch $pkgname-cgiserv::noarch"
+source="$pkgname-$pkgver.tar.gz::https://github.com/moosefs/moosefs/archive/v$pkgver.tar.gz
+	$pkgname-cgiserv.initd
+	$pkgname-cgiserv.confd
+	$pkgname-master.initd
+	$pkgname-metalogger.initd
+	$pkgname-chunkserver.initd"
+
+build() {
+	# Configure and remove systemd stuff
+	./configure --prefix=/usr \
+		--with-systemdsystemunitdir=no \
+		--sysconfdir=/etc \
+		--with-default-user=mfs \
+		--with-default-group=mfs \
+		--disable-dependency-tracking \
+		--disable-silent-rules \
+		--mandir=/usr/share/man \
+		--infodir=/usr/share/info \
+		--datadir=/usr/share \
+		--localstatedir=/var/lib \
+		--libdir=/usr/lib64
+	make
+}
+
+prepare() {
+	autoreconf -fi
+}
```

Comment:
nitpick, but defining prepare before build makes more sense to me

Reply:
moved:)